docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -719,6 +720,21 @@ testcafe ie my-tests --page-load-timeout 0
 
 *Related configuration file property*: [pageLoadTimeout](configuration-file.md#pageloadtimeout).
 
+### --browser-init-timeout \<ms\>
+
+Specifies the time (in milliseconds) for every of the specified browsers to connect and report that it is ready to test. If one or more of the browsers fail to connect within the specified period, an error is thrown.
+
+```sh
+testcafe chrome my-tests --browser-init-timeout 40000
```

Comment:
It can be helpful to describe this example a little bit. Like here we changed the default timeout of 2 minutes for Chrome, so TestCafe will dedicate 40 seconds for the browser to establish a connection. Also I think it would be better to use the timeout greater than 40 sec, because probably the common user scenario would be increasing the default timeout (e.g. 180000 ms).

Reply:
CLI:
>In this example, the timeout for a local browser is increased. The browser has three minutes to connect before TestCafe throws an error.

Config:
> In this example, the timeout for local and remote browsers is three minutes. In this run, all browsers have to connect within this time before TestCafe throws an error.

Reply:
It's good, but I think in the first one (CLI), we can change "a local browser" to "Chrome" to be more specific. Because "--browser-init-timeout" will change the timeout both for local and remote browsers.

Reply:
>In this example, browser timeout for Chrome is increased. The browser has three minutes to initialize before TestCafe throws an error.